packages/stark-ui/src/modules/minimap/components/minimap.component.ts

```diff
@@ -0,0 +1,173 @@
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import {
+	Component,
+	EventEmitter,
+	Inject,
+	Input,
+	NgZone,
+	OnChanges,
+	OnDestroy,
+	OnInit,
+	Output,
+	SimpleChanges,
+	ViewEncapsulation
+} from "@angular/core";
+import { StarkDOMUtil } from "@nationalbankbelgium/stark-ui";
+import { StarkMinimapItemProperties } from "./item-properties.intf";
+
+/**
+ * @ignore
+ */
+const _isEqual: Function = require("lodash/isEqual");
+
+/**
+ * @ignore
+ */
+const _cloneDeep: Function = require("lodash/cloneDeep");
+
+export type StarkMinimapComponentMode = "compact";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-minimap";
+
+/**
+ * Component to display a minimap which permits to toggle the visibility of passed elements.
+ * The minimap shows the label of the elements to display with a checkbox to enable/disable the visibility:
+ *
+ * @param string -mode Desired layout or flavour:
+ *
+ *    - compact: displayed in a compact form. There is only a button to trigger the dropdown.
+ *    - default: basic implementation with everything
+ *
+ * @param StarkMinimapItemProperties[] - items Array of StarkMinimapItemProperties objects which define the items to display in the minimap.
```

Comment:
Now we put all the JSDocs in the property they belong to... so this one should be at the `items` property